licenses/Apache-2.0.lre

```diff
@@ -279,17 +298,28 @@ third-party archives.
 	Copyright __20__
 
 	Licensed under the Apache License, Version 2.0 (the "License");
-	you may not use this file except in compliance with the License.
+	you may not use __1__ file
+	(( except || expect ))
+	in compliance with the License.
 	You may obtain a copy of the License at
 
 	http:/www.apache.org/licenses/LICENSE-2.0
 
 	Unless required by applicable law or agreed to in writing, software
 	distributed under the License is distributed on an "AS IS" BASIS,
 	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-
+	
 	See the License for the specific language governing permissions and
 	limitations under the License.
 ))??
 
 ))
+
+//** from github.com/jsccast/rocksdb@v0.0.0-20150219174706-b65d32cc6e76/LICENSE **//
```

Comment:
I don't believe this is appropriate - this is really a second, separate license that should be recognized separately. I suggest adding to MIT.lre:

```
{{define "MIT-NoDisclaimer.lre"}}
{{template "mit-grant"}}
{{template "mit-conditions"}}
{{end}}
```

Reply:
Removed.